.github/workflows/nightly.yml

```diff
@@ -14,25 +14,36 @@ jobs:
       uses: actions/checkout@master
       with:
         submodules: 'recursive'
-    - name: Make Solution
+    - name: Create Apps\node_modules directories
       run: mkdir Apps\node_modules &&
         mkdir Apps\node_modules\babylonjs &&
         mkdir Apps\node_modules\babylonjs-gui &&
         mkdir Apps\node_modules\babylonjs-loaders &&
-        mkdir Apps\node_modules\babylonjs-materials &&
-        mkdir buildWin32_x64 && 
-        cd buildWin32_x64 &&
-        cmake -G "Visual Studio 16 2019" -A x64 -DBGFX_CONFIG_MEMORY_TRACKING=ON -DBGFX_CONFIG_DEBUG=ON ..
-    - name: Add msbuild to PATH
-      uses: microsoft/setup-msbuild@v1.0.2
+        mkdir Apps\node_modules\babylonjs-materials
```

Comment:
I just looked this up. Curl has a `--create-dirs` option. That should make this easier.

Reply:
this should now be fixed